packages/components/file-card/src/file-card.js

```diff
@@ -0,0 +1,357 @@
+													transform="translate(258.000000, 18.000000)"
+												>
+													<path
+														d="M12.1590909,17.3636364 C11.9999992,17.3636364 11.8553726,17.2840917 11.7252066,17.125 L7.34297521,11.571281 C7.25619791,11.4555779 7.21280992,11.3615706 7.21280992,11.2892562 C7.21280992,11.1012387 7.35020524,11.0072314 7.625,11.0072314 L10.7272727,11.0072314 L10.7272727,4 L13.5909091,4 L13.5909091,11.0072314 L16.6931818,11.0072314 C16.9679766,11.0072314 17.1053719,11.1012387 17.1053719,11.2892562 C17.1053719,11.3615706 17.0619839,11.4555779 16.9752066,11.571281 L12.5929752,17.125 C12.4772721,17.2840917 12.3326455,17.3636364 12.1590909,17.3636364 Z M18.8409091,17.8409091 C18.971075,17.8409091 19.0831607,17.8879128 19.1771694,17.9819215 C19.2711782,18.0759302 19.3181818,18.1880159 19.3181818,18.3181818 L19.3181818,19.2727273 C19.3181818,19.4028932 19.2711782,19.5149789 19.1771694,19.6089876 C19.0831607,19.7029963 18.971075,19.75 18.8409091,19.75 L5.47727273,19.75 C5.34710679,19.75 5.23502113,19.7029963 5.1410124,19.6089876 C5.04700366,19.5149789 5,19.4028932 5,19.2727273 L5,18.3181818 C5,18.1880159 5.04700366,18.0759302 5.1410124,17.9819215 C5.23502113,17.8879128 5.34710679,17.8409091 5.47727273,17.8409091 L18.8409091,17.8409091 Z"
+														id="Icon"
+													></path>
+												</g>
+											</g>
+										</g>
+									</g>
+								</g>
+							</g>
+						</svg>
+					</span>
+				</div>
+			`;
+		}
+
+		get fileInformation() {
+			if (this.state === 'download') {
+				const fileExtension = this.fileObject.name
+					.split('.')
+					.pop()
+					.toUpperCase();
+
+				return html`
+					<div class="file-information">
+						<ts-file-size size="${this.fileObject.size}"></ts-file-size>
+						<ts-typography
+							text="${`| ${fileExtension} Document`}"
+							variant="subtitle"
+						></ts-typography>
+					</div>
+				`;
+			}
+		}
+
+		get actionMessage() {
+			return html`
+				<ts-typography
+					class="remove-action-message"
+					text="Remove"
+					color="action"
+					variant="subtitle"
+				></ts-typography>
+				<ts-typography
+					class="download-action-message"
+					text="Download"
+					color="action"
+					variant="subtitle"
+				></ts-typography>
+			`;
+		}
+
+		render() {
+			return html`
+				<div
+					?data-removable="${this.removable}"
+					class="file-card-wrapper state-${this.state}"
+				>
+					<ts-card
+						type="${this.cardType}"
+						orientation="horizontal"
+						?rtl="${this.rtl}"
+						data-size="${this.size}"
+					>
+						${this.fileIcon}
+						<div class="content">
+							<ts-typography
+								text="${this.fileObject.name}"
+								class="file-name"
+								variant="title"
+								no-wrap
+							></ts-typography>
+							${this.progressBar} ${this.fileInformation} ${this.errorMessages}
+							${this.actionMessage}
+						</div>
+						${this.actionIcons}
+					</ts-card>
+				</div>
+			`;
+		}
+
+		firstUpdated(_changedProperties) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "_changedProperties". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)